### DISCUSSION TITLE
Failure on Backend connector when sending long query string (delete/copy multiple selection)

### DISCUSSION THREAD

**Discussion by trannamtrung1st** [Studio-42/elFinder#3344] [Q&A]
- When I try to delete multiple files (around 50), the query string which is sent to backend is too long for web server to handle. It is a bit inconvenient. Can we consider changing the API to use request body instead?

**Comment by nao-pon** [comment]
@trannamtrung1st You can optionally choose POST communication.

- https://github.com/Studio-42/elFinder/wiki/Client-configuration-options-2.1#requesttype

**Reply by trannamtrung1st** [reply]
Many thanks. I missed that one.
